src-v2/Not/Not.Blazor/TM/Models/MultiSelectListModel.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.ComponentModel;
+using System.Linq.Expressions;
+using System.Reflection;
+
+namespace Not.Blazor.TM.Models;
+
+public class MultiSelectListModel
+{
+    public MultiSelectListModel(string value, string description)
+    {
+        Value = value;
+        Description = description;
+    }
+
+    public string Value { get; set; }
+    public string Description { get; set; }
+
+    public static IEnumerable<MultiSelectListModel> MultiSelectList<T>()
+    {   
+        var enumValues = Enum.GetValues(typeof(T)).Cast<Enum>();
+        var selectItems = enumValues.Select(s => new MultiSelectListModel(s.ToString(), EnumHelper.GetDescription(s)));
```

Comment:
`EnumHelper.GetDescription(s)`  can be  `s.GetDescription()`, because GetDescription is defined as an extension method.